contracts/Prover.sol

```diff
@@ -287,11 +297,11 @@ contract Prover is SimpleProver {
             stateRoot: bytes32(RLPReader.readBytes(RLPReader.readList(rlpEncodedBlockData)[3]))
         });
         BlockProof memory existingBlockProof = provenStates[settlementChainId];
-        if (existingBlockProof.blockNumber < blockProof.blockNumber) {
+        if ((existingBlockProof.blockNumber + SETTLEMENT_BLOCKS_DELAY) < blockProof.blockNumber) {
```

Comment:
are these parentheses necessary? not sure but thought id call it out. same on line 304